### DISCUSSION TITLE
Cloning and installing

### DISCUSSION THREAD

**Discussion by adityabanerjee992** [lucky-media/cloud#12] [General]
Hi, I have purchased this starter kit and I was wondering, how could I clone it on my system and start installing it? I am using plain docker and not using sail. So I was wondering if I could clone it and then run composer install? Please guide. Thank you.

**Comment by lokmanm** [comment]
Hi @adityabanerjee992,

If you have purchased it you can install it when creating a new Statamic project. You will be prompted to enter a one-time license or purchase one when you scaffold the project.

More info can be found [here](https://statamic.dev/starter-kits/installing-a-starter-kit#paid).

Thanks
